src/java.base/share/classes/sun/security/util/Debug.java

```diff
@@ -290,32 +290,15 @@ public void println(String prefix, String message)
     private String extraInfo() {
         String retString = "";
         if (printThreadDetails) {
-            retString = toHexString(Thread.currentThread().threadId()) + "|" +
-                    Thread.currentThread().getName() + "|" + formatCaller();
+            retString = "0x" + Long.toHexString(
+                    Thread.currentThread().threadId()).toUpperCase(Locale.ROOT) +
```

Comment:
mmhh.. i can understand the desire to unify casing, but maybe changing the other logs to use javas default casing for hex (instead of changing it here) makes the code faster and better to read?
(Not sure if StringBuilder optimization (including starting with [) would be worth the complication)